main/motion_model.h

```diff
@@ -0,0 +1,70 @@
+#ifndef MAIN_MOTION_MODEL_H
+#define MAIN_MOTION_MODEL_H
+
+#include <maze_solver/direction.h>
+#include <misc_utils/angle.h>
+#include <misc_utils/physical_size.h>
+#include <misc_utils/strongly_typed.h>
+
+#include "position.h"
+
+#include <Eigen/Dense>
+
+#include <chrono>
+
+namespace micromouse
+{
+
+static constexpr auto pos_dimension = 3;
+using PosJacobian = Eigen::Matrix<float, pos_dimension, pos_dimension>;
+using seconds = std::chrono::duration<float, std::ratio<1>>;
+static constexpr auto distance_between_wheels = 10.1e-2_m;
```

Comment:
Why use `10.1e-2`? It's harder to read...

Either

```suggestion
static constexpr auto distance_between_wheels = 0.101_m;
```

Or

```suggestion
static constexpr auto distance_between_wheels = unit_cast<meters>(10.1_cm);
```

Or

```suggestion
static constexpr auto distance_between_wheels = unit_cast<meters>(101_mm);
```